### PR TITLE
Remove overrides

### PR DESCRIPTION
**Pull request by marco-c**
In debug mode, it's still possible to override methods by defining natives (for example if you want to override an obfuscated method to understand some piece of code).

### CODE REVIEW

File: vm/parser.ts [hunk 1]
```diff
@@ -804,6 +804,11 @@ module J2ME {
     }
 
     get isNative(): boolean {
+      if (!release) {
+        if (Native[this.implKey]) {
+          return true;
+        }
+      }
```
Comment (mykmelez) [inline vm/parser.ts]: Nit: README.md should describe how to use this to override non-native methods with native implementations.

**Comment by mykmelez** [comment]
Looks good. And it shouldn't affect perf, since overrides and natives are treated almost identically by the VM. But it'd be useful to confirm that.

**Comment by marco-c** [comment]
|      Test     | Baseline Mean |   Mean   |   +/-  |   %   |       P       |    Min   |    Max   | 
|:--------------|--------------:|---------:|-------:|------:|--------------:|---------:|---------:|
| startupTime   |       1,157ms |  1,149ms |   -8ms | -0.71 | INSIGNIFICANT |    961ms |  1,425ms | 
| totalSize     |      27,567kb | 27,456kb | -111kb | -0.40 | INSIGNIFICANT | 25,918kb | 28,611kb | 
| domSize       |         173kb |    173kb |    0kb | -0.01 | INSIGNIFICANT |    173kb |    173kb | 
| styleSize     |         558kb |    558kb |    0kb |  0.01 | INSIGNIFICANT |    558kb |    560kb | 
| jsObjectsSize |      16,925kb | 16,967kb |   43kb |  0.25 |         WORSE | 16,829kb | 17,374kb | 
| jsStringsSize |         786kb |    779kb |   -7kb | -0.88 |        BETTER |    674kb |    783kb | 
| jsOtherSize   |       8,665kb |  8,519kb | -145kb | -1.68 | INSIGNIFICANT |  7,158kb |  9,284kb | 
| otherSize     |         460kb |    458kb |   -1kb | -0.30 | INSIGNIFICANT |    456kb |    474kb |

File: vm/parser.ts (hunk 1 above)
Comment (marco-c) [inline vm/parser.ts]: Added a description in https://github.com/marco-c/j2me.js/commit/c6f9ebfaf872f9368ca23e994fc41dfbc5669b14.

**Pull request merged**